itou/www/stats/views.py

```diff
@@ -255,7 +259,7 @@ def stats_cd(request):
     return render_stats(request=request, context=context, params=params)
 
 
-def render_stats_pe(request, page_title):
+def render_stats_pe(request, page_title, extra_params={}):
```

Comment:
attention, ici et dans `render_stats_dgefp` ilfaut éviter de mettre comme paramètre par défaut une valeur mutable.

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
Ah mais c'est pas la première fois que je fais cette erreur, décidément ça rentre pas dans ma tête. Cette fois je vais laisser un commentaire.